app/src/js/components/Pdr/pdr.js

```diff
@@ -256,4 +239,7 @@ PDR.propTypes = {
   history: PropTypes.object
 };
 
-export default withRouter(connect(state => state)(PDR));
+export default withRouter(
+  connect((state) => ({
+    pdrs: state.pdrs
```

Comment:
Individual PDR page is not rendering and I believe it's because we are not including `collections`, `granules`, and `logs` as part of the state we're passing to the component.

Reply:
Sorry I missed that.  Thanks!